article/sections/_2_graph_importance.tex

```diff
@@ -1,3 +1,183 @@
+cause an increase more transit oriented developments and economic activity.
+
+Policymakers base their analyses and conclusions on hypotheses or beliefs of how the world operates.
+In other words, the data analysis is based on a certain belief about the data generating process.
+Yet, policymakers often do not present their beliefs about such data generating process.
+As a result, these proposals maintain an obscure representation of how the policy or project will achieve their desired goals. 
+
+Directed Acyclic Graphs (DAGs) allow one to clearly encode their assumptions about the data generating process and the problem at hand.
+Researchers and practitioners have made use of DAGs in fields such as epidemiology, scheduling, and network structures and have found them to be practical.
+
+%% TODO: maybe insert some references of where DAGs were used?.
+
+DAGs could show to be very useful in addressing transportation policy questions, similar to the ones presented above.
+\citet{brathwaite_2018_causal} have proposed a framework illustrating how practitioners and researchers can use DAGs
+ to answer such transportation modeling questions in a causal context.
+\citet{brathwaite_2018_causal}, however, have not shown an empirical application of their framework and how it results
+ in different conclusions when compared to traditional modeling approaches.
+
+In this section, we will present an example illustrating the importance of using DAGs in transportation demand modeling efforts. 
+More specifically, we will illustrate how different assumptions about the data generating process result in different conclusions.
+We approach this exercise by building up on \citet{brathwaite_2018_causal} work by going through 
+an empirical exercise of a simplified transportation modeling problem.
+Before going any further, we would like to note that this example is for
+illustrative purposes and does not reflect the complexities of a typical transportation choice modeling problem.
+
+Let us assume that a company wants to reduce its workforce carbon footprint by moving its employees closer to their campus. 
+This action could lead to more employees not driving to work.
+We would like to forecast how such an intervention would change the share of employees driving to work.
+We model this travel mode choice problem based on a dataset from \citet{brathwaite_asymmetric}.
+This dataset is based on the 2012 California Household Travel Survey (CHTS).
+The dataset contains approximately 4000 home-based school or work tours made by approximately 3850 individuals in the California Bay area.
+Readers interested in a more detailed description of the dataset can refer to \citet{brathwaite_asymmetric}.
+Our goal in this section is not to recover the causal effect of the proposed intervention.
+We aim to only show how different DAGs would result in different conclusions about the impact of the proposed intervention.
```

Comment:
I think it'd be a good idea to move these two sentences to be the last two sentences of the previous paragraph.
```suggestion
Indeed, our goal in this section is not to recover the causal effect of the proposed intervention.
Instead, we aim to show how different DAGs result in different conclusions about the effect of the proposed intervention.
```